config/settings/base.py

```diff
@@ -0,0 +1,127 @@
+import os
+from dotenv import load_dotenv
+
+
+def reduce_path(file_name, times=3):
+    """Return directory path"""
+    result = os.path.realpath(file_name)
+    for _ in range(times):
+        result = os.path.dirname(result)
+    return result
+
+
+def rel_path(*x):
+    """Handling file paths"""
+    return os.path.abspath(os.path.join(BASE_DIR, *x))
+
+
+# Project directory
+BASE_DIR = reduce_path(__file__)
```

Comment:
Specify the number of times explicitly
```suggestion
BASE_DIR = reduce_path(__file__, times=3)
```